client/lib/plans/constants.js

```diff
@@ -208,8 +207,6 @@ export const PLANS_LIST = {
 			isEnabled( 'publicize-scheduling' ) && FEATURE_REPUBLICIZE_SCHEDULING,
 			FEATURE_BUSINESS_ONBOARDING,
 			FEATURE_ADVANCED_SEO,
-			isATEnabledForCurrentSite() && FEATURE_UPLOAD_PLUGINS,
-			isATEnabledForCurrentSite() && FEATURE_UPLOAD_THEMES,
```

Comment:
Updated in ce87dfe This changes behavior slightly so any new user can see this.

You should see the feature if we set `localStorage.ABTests = JSON.stringify( {"automatedTransfer2_20170316": "enabled"} )` then visit http://calypso.localhost:3000/plans/:siteId: